routes/handler.go

```diff
@@ -0,0 +1,19 @@
+package routes
+
+import (
+	"github.com/gorilla/mux"
+
+	"go-rest-api/controllers"
+)
+
+const VERSION = "/api/v1"
+
+//handler requests
+func HandleRequests() *mux.Router {
```

Comment:
This function could be named `NewRouter` or `GetRouter`, since that's what it seems to be doing.